gitleaks.toml

```diff
@@ -0,0 +1,8 @@
+title = "MBI Scan"
+# Only using custom rules
+useDefault = false
```

Comment:
I think we want to keep the defaults enabled to continue scanning for other secrets on commit.

Reply:
Reverted this in most recent version, closing